### PR TITLE
Conformance test for Spec Change in Section 3.8.2.2 parameter Declaration and 6.4 Destructuring parameter declarations

### PR DESCRIPTION
**Pull request by yuit**
No just the update in those two section. See this PR #1878 for the spec change

### CODE REVIEW

**Comment by DanielRosenwasser** [comment]
In the future, can you link to the relevant sections and/or give details on what updates you're testing with respect to these sections?

Are you writing tests for *all* of [3.8.2.2](https://github.com/Microsoft/TypeScript/blob/master/doc/spec.md#3822-parameter-list), and [6.4](https://github.com/Microsoft/TypeScript/blob/master/doc/spec.md#6.4)?

**Comment by yuit** [comment]
No just the update in those two section. See this PR #1878 for the spec change

File: src/compiler/diagnosticInformationMap.generated.ts [hunk 1]
```diff
@@ -1,6 +1,5 @@
 // <auto-generated />
 /// <reference path="types.ts" />
-/* @internal */
```
Comment (DanielRosenwasser) [inline src/compiler/diagnosticInformationMap.generated.ts]: Remove this change, we want this.

Comment (DanielRosenwasser) [inline src/compiler/diagnosticInformationMap.generated.ts]: Or, merge in from master.

**Comment by DanielRosenwasser** [comment]
Is there any reason you chose ES6 targets for 2 of the tests?

**Pull request merged**